app/services/moment.js

```diff
@@ -2,5 +2,5 @@ import MomentService from 'ember-moment/services/moment';
 import config from '../config/environment';
 
 export default class MomentOverride extends MomentService {
-  defaultFormat = config.moment.outputFormat;
+  defaultFormat = config.moment && config.moment.outputFormat || null;
```

Comment:
The previous usage of `get` would have fallen back to `undefined` instead; is there a benefit to using `null` instead?

If `undefined` as a fall-back is fine, I think optional chaining makes this a little nicer to read

```suggestion
  defaultFormat = config.moment?.outputFormat;
```

This would mirror what `Ember.get` was doing previously

Reply:
Huh, I thought `Ember.get` fell back to `undefined` but I see that's wrong!

If we _did_ want to use a more modern approach we could do this

```suggestion
  defaultFormat = config.moment?.outputFormat ?? null;
```

But I'm not sure that using _that_ syntax specifically is more important that fixing this bug

Reply:
I'm still in the habit of avoiding optional chaining for some reason, need to break myself of that reflex!